src/Symfony/Bundle/FrameworkBundle/Resources/config/http_client.php

```diff
@@ -49,13 +50,20 @@
 
         ->alias(ClientInterface::class, 'psr18.http_client')
 
-        ->set(\Http\Client\HttpClient::class, HttplugClient::class)
+        ->set(HttpAsyncClient::class, HttplugClient::class)
```

Comment:
let's name this one `httplug.http_client` and define an alias for the interface (+ update FrameworkExtension accordingly)